### DISCUSSION TITLE
Virtuemart payment plugin request

### DISCUSSION THREAD

**Discussion by subn3t1** [btcpayserver/btcpayserver#3498] [Plugins & integrations]
Hi!
We would love to see a Virtuemart integration plugin for the self-hosted BTCPay Server, the same way as there are for WooCommerce and all the others.

Thanks,
Stefan

[the remark this answers is not in the document]
**Reply by subn3t1** [reply]
Dear poor11,

I do not understand your reply.

Kind Regards,
Stefan

[the remark this answers is not in the document]
**Reply by pavlenex** [reply]
https://docs.btcpayserver.org/VirtueMart/
